tor/core/inbox.py

```diff
@@ -113,6 +114,15 @@ def process_reply(reply, config):
             reply.mark_read()
             return
 
+        if (
+            '&#32;' in r_body or
+            'reddit/wiki/index)' in r_body or  # the end of the footer
+            'image transcription' in r_body
+        ):
```

Comment:
Why not reuse `footer_check` from `tor.core.validation`? That should work just the same here.
I like also checking for "image transcription" since people are bad at reading.

Reply:
Also, and this is actually quite important: move this check up a few places. A transcription could include one of the other trigger words, especially "thank" could be problematic.

Reply:
1) why not use `footer_check`: because... yeah no that's a good idea

2) yes, definitely will move it up